pom.xml

```diff
@@ -47,7 +47,7 @@
     <java.level>8</java.level>
     <jenkins.version>2.138.4</jenkins.version>
     <no-test-jar>false</no-test-jar>
-    <surefire.rerunFailingTestsCount>1</surefire.rerunFailingTestsCount>
+    <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
```

Comment:
@carlossg was there a flaky test we were using the rerun option for?

Reply:
there have been some flaky tests in the past, but I'd rather avoid rerunning and fix them, and we have 2 different builds happening